tpp/tpp.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Token.SDK.Net.Tpp</AssemblyName>
+    <RootNamespace>Tokenio</RootNamespace>
+    <Copyright>Copyright 2019 Token, Inc</Copyright>
+    <Description>C# SDK for interacting with the Token System</Description>
+    <Company>Token, Inc</Company>
+    <Authors>Token, Inc</Authors>
+    <Version>2.0.0-beta1</Version>

```

Comment:
1.0.0-beta1

Reply:
Getting below error after changing the version to 1.0.0-beta1

System.AggregateException : One or more errors occurred. (Status(StatusCode=FailedPrecondition, Detail="Unsupported client version 1.0.0. Please upgrade your csharp SDK."))
---- Grpc.Core.RpcException : Status(StatusCode=FailedPrecondition, Detail="Unsupported client version 1.0.0. Please upgrade your csharp SDK.")